js-miniapp-sample/src/pages/gifs.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import { Typography, CardContent, CardMedia, Grid } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+import GreyCard from '../components/GreyCard';
+const useStyles = makeStyles((theme) => ({
+  grid: {
+    position: 'relative',
+    top: '25%',
+  },
+  typography: {
+    marginTop: '1rem',
+  },
+}));
+
+function GIFComponent() {
+  const classes = useStyles();
+  const images = [
+    {
+      label: 'Loop Count: Once',
+      iconSrc: require('../assets/images/gif/road.gif'),
+      altLabel: 'road',
+    },
+    {
+      label: 'Loop Count: Infinite',
+      iconSrc: require('../assets/images/gif/road_infinite.gif'),
+      altLabel: 'infinite_road',
+    },
+    {
+      label: 'Loop Count: Infinite (WebP)',
+      // $FlowFixMe
+      iconSrc: require('../assets/images/webp/road.webp'),
+      altLabel: 'infinite_road_webp',
+    },
+  ];
+  return (
+    <Grid
+      container
+      direction="column"
+      justify="flex-start"
+      alignItems="flex-start"
```

Comment:
I don't know if we can make it become center in mobile webview. I tried with `center`, it's okay for web but in android webview it does not change.

Reply:
@minh-rakuten fixed